deploy/releases/0.5.0/readme.adoc

```diff
@@ -69,4 +69,9 @@ _Deleting the CRD will also delete all `RuntimeComponent` in the cluster_
 * Knative support is limited. Values specified for `autoscaling`, `resources` and `replicas` parameters would not apply for Knative when enabled using `createKnativeService` parameter.
 * The auto-creation of an application definition by kAppNav is not supported when Knative is enabled.
 * Monitoring feature does not support integration with Knative Service. Prometheus Operator is required to use ServiceMonitor.
-* After the initial deployment of `RuntimeComponent`, any changes to its labels would be applied only when one of the parameters from `spec` is updated.
\ No newline at end of file
+* After the initial deployment of `RuntimeComponent`, any changes to its labels would be applied only when one of the parameters from `spec` is updated.
+
+== Known Issues
+
+* Operator can crash when creating Ingress resource if `spec.expose` is `true` and `spec.route` is not provided. 
+Possible fixes are either to disable Ingress `spec.expose` is `false` or add `spec.route: {}`
```

Comment:
Please change _disable Ingress `spec.expose` is `false`_ --> _disable Ingress by setting `spec.expose` to `false`_